cmd/geth/main.go

```diff
@@ -440,6 +440,7 @@ func startNode(ctx *cli.Context, stack *node.Node, backend ethapi.Backend) {
 		// Set the gas price to the limits from the CLI and start mining
 		gasprice := utils.GlobalBig(ctx, utils.MinerGasPriceFlag.Name)
 		ethBackend.TxPool().SetGasPrice(gasprice)
+		ethBackend.TxPool().DisableTxLifecycles()
```

Comment:
the feature is closed by default? how to enable it?

Reply:
signer disable this feature, so when `--mine` represents signer node, disable it

Reply:
got it.